package.json

```diff
@@ -18,7 +18,7 @@
     "clean": "lerna clean",
     "bs": "lerna bootstrap",
     "packages": "run-s packages:*",
-    "packages:fix": "lerna run fix",
+    "packages:fix": "lerna run fix && echo",
```

Comment:
This is a hack, I had to do it because lint-staged was passing the file names as argument to `fix` script in packages, and it was breaking them, causing a false error.

Reply:
I have felt this pain before 😅